redis/README.md

```diff
@@ -0,0 +1,64 @@
+# Redis Connectors
+
+Hazelcast Jet connector for various [Redis](https://redis.io/) data structures which enables Hazelcast Jet pipelines to 
+read/write data points from/to them:
+
+- [Streams](streams.md)
+- [Sorted Set](sorted_set.md)
+- [Hash](hash.md)
+
+## Prerequisites for all connectors
+
+### Redis 
+
+First thing you will need is a Redis installation. Check out the [Redis Quick Start](https://redis.io/topics/quickstart)
+guide on how to do this. 
+
+For development, the fastest way to get it working is to use [Homebrew](https://brew.sh/):
+- `$ brew install redis` installs it
+- `$ brew info redis` can check the installed version
+- `$ brew services start redis` starts it as a service
+- `$ redis-cli ping` can be used to make sure it's running 
+- `$ brew services stop redis` stops the service
+- `$ brew uninstall redis` uninstalls it and its files 
+
+### Java client
+
+Redis Connector artifacts are published on the Maven repositories. The connectors in turn also have a dependency of
+their own on the [Lettuce](https://github.com/lettuce-io/lettuce-core) Redis Java client, so that jar must also be
+on the classpath. 
+
+Add the following lines to your pom.xml:
+
+```
+<dependency>
+    <groupId>com.hazelcast.jet.contrib</groupId>
+    <artifactId>redis</artifactId>
+    <version>${version}</version>
+</dependency>
+
+<dependency>
```

Comment:
I don't think they need to explicitly add redis client dependency, aren't they added to the classpath via our pom.xml ?

Reply:
yes, it is not necessary while using maven or gradle. maybe we should mention that it should be on the class-path while running standalone